_posts/2016-07-31-this-week-in-rustfest-8-tickets-and-talks.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+title: "This Week in RustFest 7: Tickets and Talks"
+socialImageSrc: '/assets/posts/twirf-8-speakers.png'
+---
+
+## Tickets will be up Monday
+
+As announced on Friday, we will have the first batch of tickets sales start tomorrow, Monday August 1st at 3pm CEST (Berlin, Amsterdam).
+
+This batch will exclusively sell the supporter and View Source combo tickets, regular tickets will be made available one week after, August 8th, at the same time. By buying a supporter ticket for 250€ (excl. VAT) you directly support the diversity efforts by the team, including directly increasing the amount of diversity and student tickets we can offer. If your company will be paying for your ticket, please consider these tickets – you'd help us and the community as a whole A LOT.
+
+Secondly View Source combo ticket, priced at 350€ (excl. VAT) for early-birds, allow you to visit [View Source Conf](https://viewsourceconf.org/berlin-2016/) in addition to RustFest. It's a 3-day single-track conference about the the future of the open web platform and related technologies that Mozilla organises earlier that same week in Berlin. Regular tickets will be available from August 8th at 125€ (excl. VAT).
```

Comment:
> Secondly, 
↑ with a comma.

also: "the View Source combo ticket ... allows you ..." 
or "View Source combo tickets ... allow you" ?